services/ui-src/src/components/cards/EntityCard/EntityCardTopSection.tsx

```diff
@@ -46,7 +46,37 @@ export const EntityCardTopSection = ({
         </>
       );
     case ModalDrawerEntityTypes.QUALITY_MEASURES:
-      return <Text sx={sx.description}>Quality Measures TODO</Text>;
+      return (
+        <>
+          <Heading as="h4" sx={sx.heading}>
+            {formattedEntityData.name}
+          </Heading>
+          <Text sx={sx.subtitle}>Measure Domain</Text>
+          <Text sx={sx.subtext}>{formattedEntityData.domain}</Text>
+          <Grid gap="normal 6" templateColumns="33% 66%">
```

Comment:
Request: We need a bit more space between the first and second columns, as shown in the attached screenshot. I had a similar issue in sanctions if you want some prior art for reference: [JSX](https://github.com/CMSgov/mdct-mcr/blob/e3b08dce00c2530cbc236149a2ceb5e73484667f/services/ui-src/src/components/cards/EntityCard/EntityCardTopSection.tsx#L30-L41) | [JSS.](https://github.com/CMSgov/mdct-mcr/blob/e3b08dce00c2530cbc236149a2ceb5e73484667f/services/ui-src/src/components/cards/EntityCard/EntityCardTopSection.tsx#L98-L100)

![Screen Shot 2022-10-20 at 18 50 15](https://user-images.githubusercontent.com/27705928/197082055-9baaf4bc-5b73-4f59-bfc9-cdd911a136d9.png)